desktop/apps/auction/components/layout/__tests__/Banner.test.js

```diff
@@ -1,10 +1,25 @@
 import renderTestComponent from 'desktop/apps/auction/__tests__/utils/renderTestComponent'
 import { test } from 'desktop/apps/auction/components/layout/Banner'
-
+import sinon from 'sinon'
 const { Banner } = test
 
 describe('auction/components/layout/Banner.test', () => {
   describe('<Banner />', () => {
+    it('tracks a click on the "Enter Live Auction" button', () => {
```

Comment:
Nice test!

Reply:
how does the `window` object work here? safe to just operate on it directly? Sorry for dragging these questions out, I was just surprised when it worked right away.

Reply:
I believe [Jest](https://facebook.github.io/jest/) provides the `window` object. In Node there's a `global` object available which is essentially the equivalent to `window` in the browser. Jest provides a mock `window` object so client-side code can run inside the Node process.